adagucserverEC/CDataPostProcessors/CDataPostProcessor_ClipMinMax.cpp

```diff
@@ -20,8 +20,8 @@ template <class T> void clipData(T *data, size_t size, double min, double max) {
   }
 }
 
-int CDPPClipMinMax::execute(CServerConfig::XMLE_DataPostProc *proc, CDataSource *dataSource, int) {
-  if (isApplicable(proc, dataSource) != CDATAPOSTPROCESSOR_RUNAFTERREADING) {
+int CDPPClipMinMax::execute(CServerConfig::XMLE_DataPostProc *proc, CDataSource *dataSource, int mode) {
+  if (isApplicable(proc, dataSource, mode) != CDATAPOSTPROCESSOR_RUNAFTERREADING, mode) {
```

Comment:
I am pretty sure this doesn't do what you want.

I think this just does:
`if (mode) {`